### PR TITLE
feat(EBS):change pip prefix into /usr/local/jina

### PR DESCRIPTION
**Pull request by niuzs-alan**

### CODE REVIEW

File: scripts/deb-systemd.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ After=network.target
 [Service]
 User=ubuntu
 WorkingDirectory=/home/ubuntu
-ExecStart=/usr/local/bin/jinad
+ExecStart=/usr/local/jina/bin/jinad
```
Comment (deepankarm) [inline scripts/deb-systemd.sh:35]: We need to start `jinad` by pasing the right workspace.
Please change this to (verify before merging)
```bash
ExecStart=/usr/local/jina/bin/jinad --workspace /usr/local/jina/tmp
```

Comment (niuzs-alan) [reply scripts/deb-systemd.sh:35]: thx

Comment (niuzs-alan) [reply scripts/deb-systemd.sh:35]: In this PR, we change pip prefix from `/usr/local` to `/usr/local/jina`, which  install binary of jinad in `/usr/local/jina/bin`

Comment (deepankarm) [reply scripts/deb-systemd.sh:35]: My comment is only for passing the custom workspace directory while starting JinaD.

Comment (niuzs-alan) [reply scripts/deb-systemd.sh:35]: OK, I get it.

**Review by deepankarm — APPROVED** [review]

**Comment by niuzs-alan** [comment]
https://github.com/jina-ai/jina/issues/2082

**Pull request merged**